### DISCUSSION TITLE
Discord.js v14 Command Handler v1.0.0

### DISCUSSION THREAD

**Discussion by MGalaCyber** [MGalaCyber/Discord.js-v14-Handler-Template#3] [General]
# Discord v14 Commands Handler Template

## What's is new?
In this update version you can get:
- [x] Support Slash Commands
-- [✅] Global
-- [❎] Private
- [x] Support Message Commands
- [x] Support Modal Commands (New)
- [x] Support Button Commands
- [x] Support Select Menu Commands
- [x] Support Context Menu Commands
- [x] Support Error Logs to your dev server
- [x] Etc

<hr /><em>This discussion was created from the release <a href='https://github.com/MGalaCyber/Discord.js-v14-Command-Handler-Template/releases/tag/v1.0.0'>Discord.js v14 Command Handler v1.0.0</a>.</em>

**Comment by ShadowGaming100** [comment]
Can you check #4
